### DISCUSSION TITLE
Does NanoMQ support Shared Subscriptions and Request-Response?

### DISCUSSION THREAD

**Discussion by pstoyanov91** [nanomq/nanomq#1726] [Q&A] [answered]
The title is all I want to ask. I can't seem to find information. The website says that it fully supports mqtt 5, but these are not yet widly implemented by all brokers. So does NanoMQ support Shared Subscriptions and Request-Response workflow?

**Comment by StargazerWayne** [comment, marked as answer]
Yes, NanoMQ supports `Shared Subscriptions` and `Request-Response`. You can check [share-sub](https://www.emqx.com/en/blog/introduction-to-mqtt5-protocol-shared-subscription) and [req-res](https://www.emqx.com/en/blog/mqtt5-request-response) for tutorial.
